### PR TITLE
#71417 remove feature flag and ensure cosmos db communication works

### PR DESCRIPTION
**Pull request by tabman83**

### CODE REVIEW

**Comment by tabman83** [comment]
I have updated the load step to use the CH config section in order to reflect the KV setup.

File: src/CaptainHook.DirectorService/Program.cs [hunk 1]
```diff
@@ -97,18 +93,6 @@ private static async Task Main()
             }
         }
 
-        /// <summary>
-        /// Configures Feature Flags based on input configuration and register the instance with the container.
-        /// </summary>
-        private static FeatureFlagsConfiguration ConfigureFeatureFlags(IConfiguration config, ContainerBuilder builder)
-        {
-            var featureFlags = new FeatureFlagsConfiguration();
```
Comment (mszymeshop) [inline src/CaptainHook.DirectorService/Program.cs:105]: `FeatureFlagsConfiguration` class can be removed now that it is no longer used.

Comment (tabman83) [reply src/CaptainHook.DirectorService/Program.cs:105]: It has been in fact removed, you are probably looking at an old commit.

File: src/CaptainHook.DirectorService/DirectorService.cs [hunk 2]
```diff
@@ -28,6 +29,7 @@ public class DirectorService : StatefulService, IDirectorServiceRemoting
         private readonly IBigBrother _bigBrother;
         private readonly IFabricClientWrapper _fabricClientWrapper;
         private readonly IReaderServicesManager _readerServicesManager;
+        private readonly ICosmosDbRepository _cosmosDbRepository;
```
Comment (chomick) [inline src/CaptainHook.DirectorService/DirectorService.cs:32]: Why do we need that in Director?

Comment (chomick) [reply src/CaptainHook.DirectorService/DirectorService.cs:32]: I get we wanted to test it intially but we will not reference that repo here, right?

Comment (tabman83) [reply src/CaptainHook.DirectorService/DirectorService.cs:32]: yeah needs to be removed

**Pull request merged**